dev.lapce.lapce.yaml

```diff
@@ -0,0 +1,47 @@
+app-id: dev.lapce.lapce
+runtime: org.freedesktop.Platform
+runtime-version: 21.08
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+command: lapce
+finish-args:
+  - --device=dri
+  - --filesystem=home
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  # Full X11-Support is needed, fallback-x11 doesn't seem to work, yet.
```

Comment:
then remove wayland.

Reply:
The weird thing is that Lapce runs native on Wayland with this configuration. If I remove `--socket=x11` or use `--socket=fallback-x11` Lapce crashes on startup.
This is definitely an upstream bug, but I'd like to keep this configuration until this is fixed.

Reply:
ok.